src/tests/stringy.spec.ts

```diff
@@ -17,6 +18,11 @@ describe('Stringy methods', () => {
       const result = nougat.isPrefix('o');
       expect(result).to.equal(false);
     });
+
+    it('This should return false for chocolate as in choco', () => {
```

Comment:
Codacy has a fix for the issue: [Replace `'This·should·return·false·for·chocolate·as·in·choco'` with `"This·should·return·false·for·chocolate·as·in·choco"`](https://app.codacy.com/gh/VictorMaria/Strings-and-Arrays/pullRequest?prid=9807711)
```suggestion
    it("This should return false for chocolate as in choco", () => {
```